docs/ocis/deployment/ocis_full.md

```diff
@@ -0,0 +1,353 @@
+  {{< /hint >}}
+
+  You are installing oCIS on a server and Traefik will obtain valid certificates for you so please remove `INSECURE=true` or set it to `false`.
+
+  Traefik will issue certificates with LetsEncrypt and therefore you must set an email address in `TRAEFIK_ACME_MAIL=`.
+
+  #### Infinite Scale Release and Version
+  By default oCIS will be started in the `latest` production version.
+  You can change it to use the oCIS rolling releases by setting `OCIS_DOCKER_IMAGE=owncloud/ocis-rolling`. This will always use the latest rolling release.
+
+  If you want to use a specific version of oCIS, set the version to a dedicated tag like `OCIS_DOCKER_TAG=5.0.1`. Available  production versions can be found on [Docker Hub Production](https://hub.docker.com/r/owncloud/ocis/tags?page=1&ordering=last_updated) and available rolling releases can be found on [Docker Hub Rolling](https://hub.docker.com/r/owncloud/ocis-rolling/tags?page=1&ordering=last_updated)
+
+  {{< hint type=info title="oCIS Releases" >}}
+  You can read more about the different oCIS releases in the [oCIS Release Lifecycle](../release_roadmap.md).
+  {{< /hint >}}
+
+  Set your domain for the oCIS frontend in `OCIS_DOMAIN=`, e.g. `OCIS_DOMAIN=ocis.owncloud.test`.
+
+  Set the initial admin user password in `ADMIN_PASSWORD=`, it defaults to `admin`.
+
+  Web Office needs a public domain for the WOPI server to be set in `WOPISERVER_DOMAIN=`, where the office suite can work on the files via the WOPI protocol.
+
+  Now it's time to set up Collabora and you need to configure the domain of Collabora in `COLLABORA_DOMAIN=`.
+
+  If you want to use the Collabora admin panel you need to set the username and password for the administrator in `COLLABORA_ADMIN_USER=` and `COLLABORA_ADMIN_PASSWORD=`.
+
+* Start the docker stack
+
+  `docker-compose up -d`
+
+* You now can visit oCIS and are able to open an office document in your browser. You may need to wait some minutes until all services are fully ready, so make sure that you try to reload the pages from time to time.
+
+## Local setup
+
+This docker stack can also be run locally. One downside is that Traefik can not obtain valid SSL certificates and therefore will create self-signed ones. This means that your browser will show scary warnings. Another downside is that you can not point DNS entries to your localhost. So you have to add static host entries to your computer.
+
+On Linux and macOS you can add them to your `/etc/hosts` file and on Windows to `C:\Windows\System32\Drivers\etc\hosts` file like this:
+
+```
+127.0.0.1 ocis.owncloud.test
+127.0.0.1 traefik.owncloud.test
+127.0.0.1 collabora.owncloud.test
+127.0.0.1 onlyoffice.owncloud.test
+127.0.0.1 wopiserver.owncloud.test
+127.0.0.1 mail.owncloud.test
+127.0.0.1 companion.owncloud.test
+127.0.0.1 minio.owncloud.test
+```
+
+After that you're ready to start the application stack:
+
+`docker-compose up -d`
```

Comment:
just in case `docker-compose up -d && docker-compose up -d` or `pull_policy: always`

Reply:
you mean pull? yes.